numba/tests/npyufunc/test_errors.py

```diff
@@ -115,6 +115,7 @@ def test_floordiv_float(self):
                                 ["divide by zero encountered",
                                  "invalid value encountered"])
 
+    @skip_m1
```

Comment:
Would be good to add a note here about the FPE warning tests being a little unreliable in general, with a link to the discussion on the M1 tracking ticket.

Reply:
I will add a comment to the definition of the skip